src/elements/Neurite.cpp

```diff
@@ -351,6 +351,26 @@ void Neurite::grow(mtPtr rnd_engine, stype current_step, double substep)
  */
 void Neurite::update_growth_cones(mtPtr rnd_engine, double substep)
 {
+    if (branching_model_->use_van_pelt_){
+        if (branching_model_->is_vanpelt_branch(rnd_engine))
```

Comment:
This checks whether a VP event occurs, right?
Then I think renaming it to something like ``van_pelt_branching_occurence``